demo/grid-columns-demos.html

```diff
@@ -148,6 +145,22 @@ <h3>Hiding Columns</h3>
               root.firstElementChild.alt = rowData.item.name.first + ' ' + rowData.item.name.last;
               root.firstElementChild.src = rowData.item.picture.thumbnail;
             };
+            
+            const contextMenu = document.querySelector('vaadin-context-menu');
+            contextMenu.listenOn = document.querySelector('vaadin-button');
+            contextMenu.openOn = 'click';
+            contextMenu.renderer = function(root) {
+              root.innerHTML  = '<div style="display: flex; flex-direction: column;"></div>';
+              columns.forEach((column, i) => {
```

Comment:
please do not use arrow functions in demos as they are not transpiled on the website and will break in IE11

Reply:
I see, I meant to ask about that. Ok I will fix this.

Reply:
Done